components/filemanager/overview.md

```diff
@@ -0,0 +1,423 @@
+---
+title: Overview
+page_title: FileManager Overview
+description: Overview of the FileManager for Blazor.
+slug: filemanager-overview
+tags: telerik,blazor,filemanager,overview
+published: True
+position: 0
+---
+
+# Blazor FileManager Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/filemanager" target="_blank">Blazor FileManager component</a> is an Explorer-like component enabling you to manage file and folders.
```

Comment:
This content can highlight the component benefits and purpose - list the 2-3 most important features here - e.g. uploading, downloading, renaming, details, etc.